basic_module/staking/src/error.rs

```diff
@@ -15,6 +15,7 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pub use crate::runtime_error::Error as RuntimeError;
+pub use crate::syntax_error::Error as SyntaxError;
```

Comment:
Would you change the title to `Implement transaction and related types for the staking module`?

Reply:
Changed